noaa/track.json

```diff
@@ -0,0 +1,28 @@
+{% import "rally.helpers" as rally with context %}
+
+{
+  "short-description": "Daily weather measurement summaries from around the globe.",
+  "description": "Indexes 10M+ weather measurement summaries from NOAA.",
```

Comment:
Oh Rally does not count the documents again but I may add this feature I've just raised https://github.com/elastic/rally/issues/296.